CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 1.2.0
+
+## Features and improvements
+* Get start-docs working on linux ([#8486](https://github.com/mapbox/mapbox-gl-js/pull/8486))
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Make requirements for text offset props more precise ([#8418](https://github.com/mapbox/mapbox-gl-js/pull/8418))
+* Get Codegen to remove extraneous newlines. ([6fdbdeb](https://github.com/mapbox/mapbox-gl-js/commit/6fdbdebd25de292ab1d42e4e6beff05fbf696d2e))
+* Update integration dependencies ([#8354](https://github.com/mapbox/mapbox-gl-js/pull/8354))
+* Expose convertFilter API in the style specification ([#8493](https://github.com/mapbox/mapbox-gl-js/pull/8493))
+* Add map-specific access tokens ([#8364](https://github.com/mapbox/mapbox-gl-js/pull/8364))
+* Explicit source options now take precedence over TileJSON ([#8232](https://github.com/mapbox/mapbox-gl-js/pull/8232)) (h/t [jingsam](https://github.com/jingsam))
+* Accommodate prefers-reduced-motion settings in browser ([#8494](https://github.com/mapbox/mapbox-gl-js/pull/8494))
+* Tilt compass as the map pitches ([#8208](https://github.com/mapbox/mapbox-gl-js/issues/8208), fixed by [#8296](https://github.com/mapbox/mapbox-gl-js/pull/8296)) (h/t [pakastin](https://github.com/pakastin))
+* Add sdk support spec section for text-radial-offset  ([#8401](https://github.com/mapbox/mapbox-gl-js/pull/8401))
+
+## Bug fixes
+* Opaque pass layers now render properly over heatmap and fill-extrusion layers ([#8440](https://github.com/mapbox/mapbox-gl-js/pull/8440))
```

Comment:
For consistency, let's keep items in the same form (i.e. "Fix rendering of ....")